go/pkg/client/cas.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	repb "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
 	log "github.com/golang/glog"
+	_ "google.golang.org/genproto/googleapis/rpc/errdetails" // the proto needs to be compiled in for UnmarshallAny.
```

Comment:
Maybe more context than "unmarshallany"? since that method isn't called in this file.

Reply:
Right, changed.